### PR TITLE
controller-gen issue fix

### PR DESCRIPTION
**Pull request by Suresh-Nakkeran**
Signed-off-by: Suresh Nakkeran <suresh.nakkeran@contractors.roche.com>

**What this PR does / why we need it**:
`make generate` fails after upgrading go 1.17
```
../../go/pkg/mod/k8s.io/kube-openapi@v0.0.0-20210421082810-95288971da7e/cmd/openapi-gen/args/args.go:24:2: istio.io/api@v0.0.0-20200715212100-dbf5277541ef requires
	k8s.io/apimachinery@v0.18.1 requires
	golang.org/x/sys@v0.0.0-20191022100944-742c48ecaeb7: missing go.sum entry; to add it:
	go mod download golang.org/x/sys
../../go/pkg/mod/k8s.io/kube-openapi@v0.0.0-20210421082810-95288971da7e/pkg/generators/extension.go:24:2: istio.io/api@v0.0.0-20200715212100-dbf5277541ef requires
	k8s.io/apimachinery@v0.18.1 requires
	golang.org/x/sys@v0.0.0-20191022100944-742c48ecaeb7: missing go.sum entry; to add it:
	go mod download golang.org/x/sys
../../go/pkg/mod/k8s.io/kube-openapi@v0.0.0-20210421082810-95288971da7e/pkg/generators/api_linter.go:29:2: istio.io/api@v0.0.0-20200715212100-dbf5277541ef requires
	k8s.io/apimachinery@v0.18.1 requires
	golang.org/x/sys@v0.0.0-20191022100944-742c48ecaeb7: missing go.sum entry; to add it:
	go mod download golang.org/x/sys
../../go/pkg/mod/k8s.io/kube-openapi@v0.0.0-20210421082810-95288971da7e/pkg/generators/config.go:25:2: istio.io/api@v0.0.0-20200715212100-dbf5277541ef requires
	k8s.io/apimachinery@v0.18.1 requires
	golang.org/x/sys@v0.0.0-20191022100944-742c48ecaeb7: missing go.sum entry; to add it:
	go mod download golang.org/x/sys
../../go/pkg/mod/k8s.io/kube-openapi@v0.0.0-20210421082810-95288971da7e/pkg/generators/api_linter.go:30:2: istio.io/api@v0.0.0-20200715212100-dbf5277541ef requires
	k8s.io/apimachinery@v0.18.1 requires
	golang.org/x/sys@v0.0.0-20191022100944-742c48ecaeb7: missing go.sum entry; to add it:
	go mod download golang.org/x/sys
```	
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1238,6 +1238,7 @@ golang.org/x/sys v0.0.0-20190916202348-b4ddaad3f8a3/go.mod h1:h1NjWce9XRLGQEsW7w
 golang.org/x/sys v0.0.0-20190924154521-2837fb4f24fe/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
 golang.org/x/sys v0.0.0-20191001151750-bb3f8db39f24/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
 golang.org/x/sys v0.0.0-20191005200804-aed5e4c7ecf9/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
+golang.org/x/sys v0.0.0-20191022100944-742c48ecaeb7/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
```
Comment (yuzisun) [inline go.sum:1242]: Do you have to add this manually? wondering why it is not generated automatically by go mod.

Comment (Suresh-Nakkeran) [reply go.sum:1242]: yes @yuzisun

Comment (yuzisun) [reply go.sum:1242]: @Suresh-Nakkeran looks like there is some behavior change in go 1.17, `go run k8s.io/kube-openapi/cmd/openapi-gen` in `hack/update-openapigen.sh` no longer pull the go mod dependency and you need to run `go get k8s.io/kube-openapi` first to update `go.mod` then `go.sum` gets updated automatically.

Comment (yuzisun) [reply go.sum:1242]: Also I think we need to generate the python SDK from the updated `swagger.json`

**Pull request merged**